Cubical/Data/List/Properties.agda

```diff
@@ -122,6 +122,25 @@ cons-inj₂ = cong safe-tail
 ¬nil≡cons : ∀ {x : A} {xs} → ¬ ([] ≡ x ∷ xs)
 ¬nil≡cons {A = A} p = lower (subst (caseList (List A) (Lift ⊥)) p [])
 
+¬snoc≡nil : ∀ {x : A} {xs} → ¬ (xs ∷ʳ x ≡ [])
+¬snoc≡nil {xs = []} contra = ¬cons≡nil contra
+¬snoc≡nil {xs = x ∷ xs} contra = ¬cons≡nil contra
+
+¬nil≡snoc : ∀ {x : A} {xs} → ¬ ([] ≡ xs ∷ʳ x)
```

Comment:
Can't you just prove this in one line using `sym` and `¬snoc≡nil`?

Reply:
Yes, you are right! This would be a better solution.